backend/src/api/permissions/permissions.api.ts

```diff
@@ -18,8 +25,17 @@ const initPermissionsApi: FastifyPluginAsync<Options> = async (
   fastify.route({
     method: HttpMethod.GET,
     url: PermissionApiPath.ROOT,
-    async handler(req, rep) {
-      const permissions = await permissionService.getAll();
+    schema: { params: paginationParamsValidationSchema },
```

Comment:
You are getting pagination values from query, so you should validate `querystring` instead of `params`
```suggestion
    schema: { querystring: paginationParamsValidationSchema },
```